addons/account/views/account_move_views.xml

```diff
@@ -632,7 +632,7 @@
                                 attrs="{'invisible': ['|', ('move_type', '!=', 'entry'), '|', ('state', '!=', 'posted'), ('payment_state', '=', 'reversed')]}"/>
                         <button name="action_reverse" string='Add Credit Note'
                                 type='object' groups="account.group_account_invoice"
-                                attrs="{'invisible': ['|', ('move_type', 'not in', ('out_invoice', 'in_invoice')), ('state', '!=', 'posted')]}"/>
+                                attrs="{'invisible': ['|', '|', ('move_type', 'not in', ('out_invoice', 'in_invoice')), ('state', '!=', 'posted'), ('amount_residual', '=', 0)]}"/>
```

Comment:
The residual is the remainder after reconciliation. Is this really what you want?
Shouldn't it be the `amount_total`?

Reply:
Yup, TSB told me the same before you did.
I will change it for amount_total but then I have to hide the radio button of the wizard because it gets disabled anyway.